applications/eradius_prometheus_collector/src/eradius_prometheus_collector.erl

```diff
@@ -252,18 +247,51 @@ get_metric_info(Name, Stat) ->
         server -> {Metrics, proplists:get_value(Metric, NasFields), RadiusMetricType}
     end.
 
-%% Helper to get a value of a server/nas metric by the given #nas_counter{} index
+%% Helper to get a value of a server/nas metric by the given #nas_counter{} or #client_counter{} index
 %% @private
-get_server_labels_and_val(MetricIdx, #nas_counter{} = Cnt) ->
+get_labels_and_val(_, {#nas_counter{} = Cnt, server}, {Name, Labels})
+  when Name =:= accounting_requests_total;
+       Name =:= accounting_responses_total ->
+    Type = proplists:get_value(acct_type, Labels),
```

Comment:
What if we will call `fetch_metric` without certain accounting_type to get all accounting_requests_total or accounting_responses_total for all accounting types?